adoc/chapters/programming_interface.adoc

```diff
@@ -17218,15 +17218,15 @@ a@
 template <access::address_space AddressSpace, access::decorated IsDecorated>
 void load(size_t offset, multi_ptr<const DataT, AddressSpace, IsDecorated> ptr)
 ----
-   a@ Loads the values at the address of [code]#ptr# offset in elements of type [code]#DataT# by [code]#NumElements * offset#, into the components of this SYCL [code]#vec#.
+   a@ Loads [code]#NumElements# elements from consecutive addresses, with the starting address determined by incrementing [code]#ptr# by [code]#offset# provided in terms of number of elements of type [code]#DataT#,  into the components of this SYCL [code]#vec#. The [code]#ptr# must be aligned to [code]#alignof(DataT)#.
```

Comment:
This is not how DPC++ behaves.  Notice how the original wording said:

> offset in elements of type `DataT` by `NumElements * offset`

I think the intent is that `offset` is multiplied by the size of the entire vector (i.e. `NumElements * offset`).  This is how DPC++ behaves.  Therefore, I think the wording should be like this instead:

> Loads `NumElements` elements into the components of `vec`.  These elements are loaded from consecutive addresses, where the starting address is computed by adding `offset * NumElements * sizeof(DataT)` bytes to the address specified by `ptr`.  The `ptr` must be aligned to `alignof(DataT)`.